app/javascript/src/utilities/ie11Utils.js

```diff
@@ -0,0 +1,3 @@
+const isBrowserIE11 = () => !!window.navigator.userAgent.match(/Trident\/7\./)
```

Comment:
use DI here for easier and isolated testing, like:

```
 const isBrowserIE11 = (win = window) => !!win.navigator.userAgent.match(/Trident\/7\./)
```

then in the test you could inject the value you want.

Reply:
Good idea, it's updated @didierofrivia !